### ISSUE TITLE
'six' upgrade issue

### ISSUE THREAD

**Issue by MaximilianR** [msiemens/PyGitUp#44]
After upgrading to six 1.10, `git up` fails. Does it really need exactly 1.9?

Full stack trace below

```
git up
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 640, in _build_master
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 941, in require
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 833, in resolve
pkg_resources.ContextualVersionConflict: (six 1.10.0 (/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages), Requirement.parse('six==1.9.0'), {'git-up'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/bin/git-up", line 5, in <module>
    from pkg_resources import load_entry_point
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 664, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 634, in _load_backward_compatible
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 3141, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 3127, in _call_aside
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 3154, in _initialize_master_working_set
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 642, in _build_master
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 655, in _build_from_requirements
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/setuptools-20.1.1-py3.5.egg/pkg_resources/__init__.py", line 828, in resolve
pkg_resources.DistributionNotFound: The 'six==1.9.0' distribution was not found and is required by git-up
```

**Comment by msiemens** [comment]
There's no particular reason why exactly `1.9` is required except that it was the latest version when the last version of PyGitUp was released. I'll update the dependencies in the next release of PyGitUp. I can't promise any release date, but I hope to get it done this months.

**Comment by msiemens** [comment]
Should be fixed now that v1.4.0 is released :)
